composer.json

```diff
@@ -46,7 +46,7 @@
             "dev-master": "3.0.x-dev"
         },
         "config-plugin": {
-            "common": "config/common.php"
```

Comment:
Wouldn't that break https://github.com/yiisoft/yii-demo/blob/master/src/Command/Router/ListCommand.php?